ledger/ledger-api-integration-tests/src/test/itsuite/scala/com/digitalasset/platform/tests/integration/ledger/api/ActiveContractsServiceIT.scala

```diff
@@ -102,15 +96,17 @@ class ActiveContractsServiceIT
       case ce @ CreatedEvent(_, _, Some(`template`), _, _, _) => ce
     }.size should equal(occurrence)
 
-  def threeCommands(ledgerId: String, commandId: String): SubmitAndWaitRequest =
+  def threeCommands(ledgerId: domain.LedgerId, commandId: String): SubmitAndWaitRequest =
     super.dummyCommands(ledgerId, commandId, "Alice").toWait
 
   private def filter = TransactionFilter(Map(config.parties.head -> Filters()))
 
   "Active Contract Set Service" when {
     "asked for active contracts" should {
       "fail with the expected status on a ledger Id mismatch" in allFixtures { context =>
-        client(context, UUID.randomUUID().toString)
+        new ActiveContractSetClient(
+          domain.LedgerId("not" + context.ledgerId.unwrap),
```

Comment:
```suggestion
          domain.LedgerId(s"not${context.ledgerId.unwrap}"),
```

Reply:
it's better to use the macro style for string concatenation

Reply:
Done everywhere.